cibuildwheel/resources/build-platforms.toml

```diff
@@ -63,7 +61,5 @@ python_configurations = [
   { identifier = "cp38-win_amd64", version = "3.8.9", arch = "64" },
   { identifier = "cp39-win32", version = "3.9.4", arch = "32" },
   { identifier = "cp39-win_amd64", version = "3.9.4", arch = "64" },
-  { identifier = "pp27-win32", version = "2.7", arch = "32", url = "https://downloads.python.org/pypy/pypy2.7-v7.3.3-win32.zip" },
-  { identifier = "pp36-win32", version = "3.6", arch = "32", url = "https://downloads.python.org/pypy/pypy3.6-v7.3.3-win32.zip" },
-  { identifier = "pp37-win32", version = "3.7", arch = "32", url = "https://downloads.python.org/pypy/pypy3.7-v7.3.3-win32.zip" },
+  { identifier = "pp37-win_amd64", version = "3.7", arch = "32", url = "https://downloads.python.org/pypy/pypy3.7-v7.3.4rc2-win64.zip" },
```

Comment:
```suggestion
  { identifier = "pp37-win_amd64", version = "3.7", arch = "64", url = "https://downloads.python.org/pypy/pypy3.7-v7.3.4rc2-win64.zip" },
```
Update arch?

Reply:
Oh, great point! So we don't use this? Or we don't check it?
Let me double check.

Reply:
Done (and added a minor extra check), thanks!